### PR TITLE
release notes for 0.17.2

### PR DESCRIPTION
**Pull request by emmanuelle**
Please tell me if I have forgotten something.

### CODE REVIEW

File: doc/release/release_0.17.rst [hunk 1]
```diff
@@ -1,7 +1,43 @@
...
+   * ``beta1`` and ``beta2`` from ``skimage.filters.frangi``
+   * ``beta1`` and ``beta2`` from ``skimage.filters.hessian``
+   * ``dtype`` from ``skimage.io.imread``
+   * ``img`` from skimage.morphology.skeletonize_3d.
+- Gallery examples were updated to suppress warnings and take into account new
+  default values in some functions (#4692 and #4676)
+
+
+
+ authors added to this release [alphabetical by first name or login]
```
Comment (sciunto) [inline doc/release/release_0.17.rst:25]: The number is missing here. Weird.

File: doc/release/release_0.17.rst [hunk 2]
```diff
@@ -1,7 +1,43 @@
...
+Announcement: scikit-image 0.17.2
 =================================
 
-We're happy to announce the release of scikit-image v0.17.0!
+We're happy to announce the release of scikit-image v0.17.2, which is a bug-fix
+release.
+
+Bug fixes
+---------
+- We made pooch an optional dependency, since it has been added as required
```
Comment (sciunto) [inline doc/release/release_0.17.rst:9]: I guess a blank line is missing here to get a green light

**Comment by emmanuelle** [comment]
Thanks @sciunto. I just merged #4698.

**Comment by emmanuelle** [comment]
@meeseeksdev backport to v0.17.x

**Pull request merged**